applications/FluidDynamicsApplication/python_scripts/apply_slip_process.py

```diff
@@ -40,6 +45,15 @@ def __init__(self, Model, settings ):
             node.SetSolutionStepValue(KratosMultiphysics.IS_STRUCTURE,0,1.0)
             node.SetValue(KratosMultiphysics.Y_WALL,0.0)
 
+        if self.navier_slip_active:
+            navier_slip_length = settings["uniform_navier_slip_length"].GetDouble()
+            for node in self.model_part.Nodes:
+                node.SetValue(KratosMultiphysics.FluidDynamicsApplication.SLIP_LENGTH, navier_slip_length)
```

Comment:
One constant value per node! This turns to a huge memory consumption to save a constant value. Why don't you save it inside the `ProcessInfo`?

Reply:
Thank you for the feedback. The idea behind this implementation was to be able to extend the functionality to spatially varying slip length values as e.g. necessary for a more complex terrain (and also tested for). Thus, I was only considering the constant value as a simplified test case for the beginning. Here, of course, it does not make sense to store the same value multiple times.